sklearn/metrics/pairwise.py

```diff
@@ -1718,7 +1718,7 @@ def additive_chi2_kernel(X, Y=None):
 
     Returns
     -------
-    kernel : ndarray of shape (n_samples_X, n_samples_Y)
+    kernel : array-like of shape (n_samples_X, n_samples_Y)
```

Comment:
I remember seeing the discussion in 1 PR, but I don't remember the outcome. Should we update the docstrings in the Array API PRs or not yet (given that the support is experimental)?

Reply:
This change was from an unrelated PR. It's gone away after a merge with `main`.

But in general we should avoid changing the type declaration in docstrings in array API PRs as long as array API dispatch is not enabled by default.